### PR TITLE
Implement Blog, Tag, Comment, Subscriber Model

### PR DESCRIPTION
**Pull request by Aju100**

### CODE REVIEW

**Comment by Aju100** [comment]
@AnjalBam Here is how I am able to pass all those test cases. Now we can accept PR.
Any review on it? @Aju100 @rajeshpandey2053 @razyesh @AnjalBam

File: blog/admin.py [hunk 1]
```diff
@@ -0,0 +1,8 @@
+from django.contrib import admin
+from .models import Category,Tag,Blog,Comment,Subscriber
+
+admin.site.register(Category)
+admin.site.register(Tag)
+admin.site.register(Blog)
+admin.site.register(Comment)
+admin.site.register(Subscriber)
```
Comment (AnjalBam) [inline blog/admin.py:8]: New Line at the end PEP-8

Comment (Aju100) [reply blog/admin.py:8]: sorry i didn't get the point

Comment (Aju100) [reply blog/admin.py:8]: @AnjalBam  I have maintain now.

File: blog/models.py [hunk 2]
```diff
@@ -1,56 +1,64 @@
...
     content = models.TextField()
-    category = models.ForeignKey(Category,on_delete = models.CASCADE)
-    STATUS_CHOICES = (('Published','published'),('Draft','draft'))
-    status = models.CharField(max_length = 10,default = 'Draft',choices = STATUS_CHOICES)
-    #user = models.ForiegnKey(User,on_delete = models.CASCADE)
+    category = models.ForeignKey(Category, on_delete=models.CASCADE)
+    STATUS_CHOICES = (('Published', 'published'), ('Draft', 'draft'))
+    status = models.CharField(max_length=10, default='Draft',
+                              choices=STATUS_CHOICES)
+    user = models.ForeignKey(User, on_delete=models.CASCADE)
```
Comment (razyesh) [inline blog/models.py:34]: related_name

File: blog/models.py [hunk 3]
```diff
@@ -1,56 +1,64 @@
...
+    updated_at = models.DateTimeField(auto_now=True)
+    is_published = models.DateTimeField(default=timezone.now)
+
     def __str__(self):
         return self.title
-    
-    def get_image_url(self,obj):
+
+    def get_image_url(self, obj):
         return obj.image.url
```
Comment (razyesh) [inline blog/models.py:46]: Auto Slug Field lagi save() method overrite gardeu

Comment (Aju100) [reply blog/models.py:46]: thank u dae

**Pull request merged**